test/functional/tests/example/example_test.py

```diff
@@ -116,3 +117,39 @@ def test_create_example_files():
             TestRun.LOGGER.info(f"Item {str(item)} - {type(item).__name__}")
     with TestRun.step("Remove file"):
         fs_utils.remove(file1.full_path, True)
+
+
+@pytest.mark.require_disk("cache1", DiskTypeSet([DiskType.hdd, DiskType.hdd4k, DiskType.sata]))
+@pytest.mark.require_disk("cache2", DiskTypeSet([DiskType.hdd, DiskType.hdd4k, DiskType.sata]))
+def test_lvm_example():
+    """
+        title: Example test using LVM API.
+        description: Create and discover Logical Volumes.
+        pass_criteria:
+          - LVM created.
+          - LVMs discovered.
+    """
+    with TestRun.step("Prepare"):
+        disk_1 = TestRun.disks['cache1']
+        disk_1.create_partitions([Size(8, Unit.GiB)] * 2)
+        test_disk_1 = disk_1.partitions[0]
+        test_disk_3 = disk_1.partitions[1]
+
+        disk_2 = TestRun.disks['cache2']
+        disk_2.create_partitions([Size(8, Unit.GiB)] * 2)
+        test_disk_2 = disk_2.partitions[0]
+        test_disk_4 = disk_2.partitions[1]
+
+    with TestRun.step("Create LVMs"):
+        lvm = Lvm.create(Size(5, Unit.GiB), [test_disk_1, test_disk_2], "lvm5")
+        lvm2 = Lvm.create(70, [test_disk_3, test_disk_4], "lvm7")
```

Comment:
You could've use `lvm1` and `lvm2` here, so you wouldn't run out of names a few lines later. ;) But nevertheless, it works fine, so I'm good with it.

Reply:
Correct, thanks! :D